test/integration/css-features/test/css-modules.test.js

```diff
@@ -58,32 +70,48 @@ describe('CSS Modules: Import Global CSS', () => {
     () => {
       const appDir = join(fixturesDir, 'module-import-global')
 
-      let stdout
-      let code
+      let appPort
+      let app
       beforeAll(async () => {
         await remove(join(appDir, '.next'))
-        ;({ code, stdout } = await nextBuild(appDir, [], {
-          stdout: true,
-        }))
+        const { code } = await nextBuild(appDir)
+        if (code !== 0) {
+          throw new Error('failed to build')
+        }
+        appPort = await findPort()
+        app = await nextStart(appDir, appPort)
       })
-
-      it('should have compiled successfully', () => {
-        expect(code).toBe(0)
-        expect(stdout).toMatch(/Compiled successfully/)
+      afterAll(async () => {
+        await killApp(app)
       })
 
       it(`should've emitted a single CSS file`, async () => {
-        const cssFolder = join(appDir, '.next/static/css')
-
-        const files = await readdir(cssFolder)
-        const cssFiles = files.filter((f) => /\.css$/.test(f))
-
-        expect(cssFiles.length).toBe(1)
-        const cssContent = await readFile(join(cssFolder, cssFiles[0]), 'utf8')
-
-        expect(
-          cssContent.replace(/\/\*.*?\*\//g, '').trim()
-        ).toMatchInlineSnapshot(`"a .styles_foo__G5630{all:initial}"`)
+        const content = await renderViaHTTP(appPort, '/')
+        const $ = cheerio.load(content)
+
+        const cssSheet = $('link[rel="stylesheet"]')
+        expect(cssSheet.length).toBe(1)
+
+        const stylesheet = cssSheet.attr('href')
+
+        const cssContent = (
+          await fetchViaHTTP(appPort, stylesheet).then((res) => res.text())
+        )
+          .replace(/\/\*.*?\*\//g, '')
+          .trim()
+
+        if (process.env.TURBOPACK) {
+          expect(cssContent.replace(/\/\*.*?\*\//g, '').trim())
+            .toMatchInlineSnapshot(`
+            "a .foo {
```

Comment:
Just noticed the behavior is different here. `@import './styles.css';` in Turbopack does not treat the imported module as CSS Modules too, while webpack does handle it as CSS modules.

Reply:
Discussed with @sokra and decided that the Turbopack behavior is more correct than the webpack behavior, as it ensures extensions deciding the type of module.